src/Engine/BasicEngine.php

```diff
@@ -42,6 +42,15 @@ public function escapeLike(string $parameter): string
         return str_replace(['%', '_'], ['\\%', '\\_'], $parameter);
     }
 
+    public function getSqlParamValue($value): string
+    {
+        if (is_null($value) or is_bool($value)) {
```

Comment:
Use `||` instead of `or` please.

Reply:
I've changed the operator (almost not doing it :P ).